luigi_pipeline/lib/model/base_mt_schema.py

```diff
@@ -0,0 +1,104 @@
+from abc import abstractmethod
+from inspect import getmembers, isfunction
+from functools import wraps
+
+
+def mt_annotation(annotation=None, fn_require=None):
```

Comment:
Interesting, I was deciding between `annotation` and `name` and thought `name` would be confusing, but I see how it could be clearer. I can change.
Yeah I thought of the `mt_row_annotation` as well to be really general, but so far I've only seen row annotations and didn't know if col should even be supported. But if it makes it clearer I can change.

Reply:
I guess row annotations as the default makes sense even if we later add `mt_column_annotation`.